core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -378,22 +378,22 @@ object TopicCommand extends Logging {
 
   def shortMessageSizeWarning(maxMessageBytes: Int): String = {
     "\n\n" +
-      "*****************************************************************************************************\n" +
-      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the consumer ***\n" +
-      "*** default. This operation is potentially dangerous. Consumers will get failures if their        ***\n" +
-      "*** fetch.message.max.bytes < the value you are using.                                            ***\n" +
-      "*****************************************************************************************************\n" +
+      "******************************************************************************************************\n" +
+      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the consumer     ***\n" +
+      "*** default. This operation is potentially dangerous. Consumers will get failures if their         ***\n" +
+      "*** max.partition.fetch.bytes < the value you are using.                                           ***\n" +
+      "******************************************************************************************************\n" +
       s"- value set here: $maxMessageBytes\n" +
-      s"- Default Consumer fetch.message.max.bytes: ${ConsumerConfig.FetchSize}\n" +
+      s"- Default Consumer max.partition.fetch.bytes: ${ConsumerConfig.FetchSize}\n" +
```

Comment:
That's a good point and make it switch to using the new consumer class. Will update shortly. Thanks.